modules/react/popup/lib/PopupBody.tsx

```diff
@@ -3,10 +3,20 @@ import * as React from 'react';
 import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
 import {Card} from '@workday/canvas-kit-react/card';
 import {usePopupModel} from './hooks';
+import {createStencil} from '@workday/canvas-kit-styling';
+import {mergeStyles} from '../../layout';
+import {system} from '@workday/canvas-tokens-web';
+
+const popupBodyStyles = createStencil({
```

Comment:
this need to be exported and `popupBodyStencil`

Reply:
Yup updated